doc/user/content/sql/create-table.md

```diff
@@ -70,6 +83,19 @@ connections. They are always created in the special `mz_temp` schema.
 Temporary tables may depend upon other temporary database objects, but non-temporary
 tables may not depend on temporary objects.
 
+### Memory usage
+
+Every table is backed by an [index](/overview/api-components/#indexes) that
+materializes all of the data in the table. Unlike sources, you cannot create an
+"unmaterialized" table. Therefore you must ensure that the data you store in
+tables fits in the amount of memory you have available on your system. Remember
+that any additional indexes or derived materialized views will count against
+your memory budget.
+
+If your dataset is too large to fit in memory, consider using an unmaterialized
+[source](/sql/create-source) instead. Then you can create defer materialization
```

Comment:
Believe the "create" here is extraneous.

Also, I don't have a rule to point to, but starting this sentence with "Then" also sounds a little off to me––but might just be a style nit. If you're interested, I offer:

> This lets you defer materialization to views on this source, which can aggregate or filter the data down to a manageable size.

Feel free to discard this, though.

Reply:
SGTM thanks!